crates/pallet-grandpa-finality-verifier/Cargo.toml

```diff
@@ -31,6 +29,8 @@ sp-trie = { git = "https://github.com/paritytech/substrate", rev="c364008a6c7da8
 [dev-dependencies]
 sp-core = { git = "https://github.com/paritytech/substrate", rev="c364008a6c7da8456e17967f55edf51e45146998" }
 sp-io = { git = "https://github.com/paritytech/substrate", rev="c364008a6c7da8456e17967f55edf51e45146998" }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", rev="c364008a6c7da8456e17967f55edf51e45146998" }
+ed25519-dalek = { version = "1.0", default-features = false, features = ["u64_backend"] }
```

Comment:
We keep dependencies in alphabetical order, please sort them.

Reply:
Should be fixed here: https://github.com/subspace/subspace/commit/018ae0750c2c0b028e70c8900a5c23b5487e5538

Reply:
Not sorted, neither in dependencies list nor in the list of features.